### ISSUE TITLE
COCO2017-yolov3-map error

### ISSUE THREAD

**Issue by QZ-cmd** [ShuangXieIrene/ssds.pytorch#78]
I use yolov3-MobilenetV1 Pre training weight to train 100 epoch，but map only 46.4%(iou=0.5)，I want to know what's wrong. thanks

**Comment by ww-zwj** [comment]
I think the author has done pre-training on coco and then directly training on VOC, so the total training epochs are only 100. If you train directly on VOC, you need to increase the total epoch and Warm_ UP_ Epochs, and increase the initial learning rate.

**Comment by QZ-cmd** [comment]
> I think the author has done pre-training on coco and then directly training on VOC, so the total training epochs are only 100. If you train directly on VOC, you need to increase the total epoch and Warm_ UP_ Epochs, and increase the initial learning rate.

Thanks for your reply
